src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -129,6 +130,14 @@ public void clear() {
         internalList.clear();
     }
 
+    /**
+     * Sorts the names of all persons in list by alphabetical order.
+     *
+     * @param comparator
```

Comment:
[Coding standard](https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html) violation! One more more of these problems detected:
1. Javadoc style format not followed.

`@param comparator` does not have any following comments, perhaps `@param comparator cannot be null.`